src/test/java/org/jabref/logic/importer/fetcher/DOABFetcherTest.java

```diff
@@ -0,0 +1,244 @@
+                "building and construction, coastal area coverage, search and rescue (SAR), water plume " +
+                "temperature measurements, aeromagnetic and archaeological surveys applications");
+        Antonios_Tsourdos.setField(StandardField.LANGUAGE, "English");
+        Antonios_Tsourdos.setField(StandardField.KEYWORDS, "UAV-Based Remote Sensing, drones, pedestrian detection and tracking as well as a range of atmospheric, UAV images, fuzzy-based hybrid control algorithms, water plume temperature measurements, aeromagnetic and archaeological surveys, geological, two-UAV communication system, agricultural, wildlife, reef, detection, multi-target localization, ecological, search and rescue (SAR), coastal area coverage, remote sensing, environmental science, segmentation in digital surface models for 3D reconstruction, location and grasping objects, augmented reality tools, building and construction, aerial robotics, noise suppression techniques, rectification for oblique images, vision-based tracking in cooperative multi-UAV systems");
+        Antonios_Tsourdos.setField(StandardField.PUBLISHER, "MDPI - Multidisciplinary Digital Publishing Institute");
+    }
+
+    @Test
+    public void TestGetName() {
+        assertEquals("DOAB", fetcher.getName());
+    }
+
+    @Test
+    public void TestPerformSearch() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("i open fire");
+        assertEquals(Collections.singletonList(David_Opal), entries);
+    }
+
+     @Test
+    public void TestPerformSearch2() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("the deliverance of open access books");
+        assertEquals(Collections.singletonList(Ronald_Snijder), entries);
+    }
+
+    @Test
+    public void TestPerformSearch3() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("Four Kingdom Motifs before and beyond the Book of Daniel");
+        assertEquals(Collections.singletonList(Andrew_Perrin), entries);
+    }
+
+    @Test
+    public void TestPerformSearch4() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("UAV Sensors for Environmental Monitoring");
+        assertEquals(Collections.singletonList(Felipe_Gonzalez), entries);
+    }
+
+    @Test
+    public void TestPerformSearch5() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("The symbiosis between information system project complexity and information system project success");
+        assertEquals(Collections.singletonList(Carl_Marnewick), entries);
+    }
+
+    @Test
+    public void TestPerformSearch6() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("UAV‐Based Remote Sensing Volume 2");
+        assertEquals(Collections.singletonList(Antonios_Tsourdos), entries);
+    }
```

Comment:
This should be converted to a parameterized test --> https://www.baeldung.com/parameterized-tests-junit-5

Reply:
> This should be converted to a parameterized test --> https://www.baeldung.com/parameterized-tests-junit-5

Hi, do you mind taking a look at the test file for the last commit, I still want to use assertEquals or something similar to it but I couldn't use it with parameterized test, any advice on that ?